package-lock.json

```diff
@@ -5828,89 +4341,59 @@
       "integrity": "sha512-baxW4Fvdz2k/iYYqEflBpRz6sulP6OskNUn+pPHJ5ZbDzYo/x639Srt4Lzb0SXSvNMDxBCzjXQIIJzsiRv24+w==",
       "dev": true,
       "requires": {
-        "archiver": "2.1.0",
+        "archiver": "2.1.1",
         "callsites": "2.0.0",
-        "chalk": "2.3.0",
+        "chalk": "2.3.2",
         "chokidar": "1.7.0",
-        "download-github-repo": "0.1.3",
+        "download-github-repo": "0.1.4",
```

Comment:
This, "download-github-repo" is the "graphcool-cli" dependency that caused the ripple effect removing the offending version of "tar"